src/main/telemetry/frsky.c

```diff
@@ -190,7 +191,14 @@ static void sendThrottleOrBatterySizeAsRpm(void)
 {
     sendDataHead(ID_RPM);
     if (ARMING_FLAG(ARMED)) {
-        serialize16(rcCommand[THROTTLE] / BLADE_NUMBER_DIVIDER);
+        if(rcCommand[THROTTLE] <= escAndServoConfig->minthrottle){
```

Comment:
I was using
```static rxConfig_t *rxConfig;```

getting confused! - been many years since I last did anything in C ! doh! - this was why I'd initially used the check against escAndServoConfig->minthrottle